gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/SyncApiReactor.java

```diff
@@ -180,7 +198,12 @@ private Completable handleRequest(final RequestExecutionContext ctx) {
             .onErrorResumeNext(t -> handleError(ctx, t))
             // Finally, end the response.
             .andThen(endResponse(ctx))
-            .doOnComplete(() -> log.debug("Response has ended"));
+            .doOnComplete(
+                () -> {
+                    pendingRequests.decrementAndGet();
+                    log.debug("Response has ended");
```

Comment:
May I ask you to remove completely the `.doOnComplete(...) ` the debug log seems useless so we can remove it.

Thx 🙂